kwc-nav-item.html

```diff
@@ -7,63 +7,84 @@
 `kwc-nav-item`
 Display a tab item in the `kwc-nav`
 
-@hero demo/hero.png
 @demo demo/index.html
 -->
 
 <dom-module id="kwc-nav-item">
     <template>
         <style>
             :host {
-                display: block;
+                display: inline-block;
             }
             :host .nav-item {
```

Comment:
The :host selector is superfluous in this context. I had it on a component because I copied the styles from the other repo, but any styles declared in the template are already scoped to that component (presumably using polymer shady-dom or shadow-dom depending on browser capabilities).

Reply:
Not the first style btw, which specifically selects the element to add styles to it, the rest of them

Reply:
there you go https://github.com/KanoComponents/kwc-nav-item/pull/8/commits/b3f0ba3848af050b50b2ba77471231a40bb0feac